_data-prepper/managing-data-prepper/configuring-data-prepper.md

```diff
@@ -100,3 +101,102 @@ usage | Yes | Bytes | Specifies the JVM heap usage at which to trip a circuit br
 check_interval | No | Duration | Specifies the time between checks of the heap size. Defaults to `500ms`.
 
+### Extension plugins
+
+Since Data Prepper 2.5, Data Prepper provides support for user configurable extension plugins. Extension plugins are shared common 
+configurations shared across pipeline plugins, i.e. [source/buffer/processor/sink]({{site.url}}{{site.baseurl}}/data-prepper/index/#concepts).
+
+#### AWS extension plugins
+
+Collection of AWS resource related extension plugins. All such plugin configuration objects are under `aws:`
+
+| Option | Required | Type   | Description                              |
+|:-------|:---------|:-------|:-----------------------------------------|
+| aws    | No       | Object | The AWS extension plugins configuration. |
+
+##### AWS secrets extension plugin
+
+Configures [AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager/latest/userguide/intro.html) extension plugin to be 
+referenced in pipeline plugin configurations, e.g.
+
+```
+extensions:
+  aws:
+    secrets:
+      <YOUR_SECRET_CONFIG_ID_1>:
+        secret_id: <YOUR_SECRET_ID_1>
+        region: <YOUR_REGION_1>
+        sts_role_arn: <YOUR_STS_ROLE_ARN_1>
+        refresh_interval: <YOUR_REFRESH_INTERVAL>
+      <YOUR_SECRET_CONFIG_ID_2>:
+        ...
+```
+
+| Option  | Required | Type   | Description                                                                                  |
+|:--------|:---------|:-------|:---------------------------------------------------------------------------------------------|
+| secrets | No       | Object | The AWS Secrets Manager extension plugin configuration. See [Secrets](#secrets) for details. |
+
+###### Secrets
+
+Multiple secrets configuration objects can be defined with unique id for each.
+
+| Option           | Required | Type     | Description                                                                                                                                                                                                                                                    |
+|:-----------------|:---------|:---------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| secret_id        | Yes      | String   | The AWS secret name or ARN.                                                                                                                                                                                                                                    |
+| region           | No       | String   | The AWS region of the secret. Defaults to `us-east-1`.                                                                                                                                                                                                         |
```

Comment:
@chenqi0805 , Does this default to `us-east-1`? I'd think it uses the AWS SDK's default. This can be the `AWS_REGION` environment variable or `us-east-1`. Do we know for sure?

Reply:
Yes. We do default to `us-east-1` the same way as the S3 DLQ in opensearch sink